setup.py

```diff
@@ -71,7 +71,7 @@
     "jax>=0.3.4",
     "numpy>=1.20.0",
     "typeguard>=2.13.3",
-    "typing_extensions>=4.2.0",
+    "typing_extensions",
```

Comment:
Please continue to specify a minimum version number. I imagine something in the `4.x.x.` series should suffice.

Reply:
Done. I chose `=3.7.4.1,<4.2.0;` to match https://github.com/explosion/confection/blob/main/setup.cfg#L33 which @slinderman says we need for colab.